modules/newrelic/keyboard.go

```diff
@@ -0,0 +1,9 @@
+package newrelic
+
+import "github.com/gdamore/tcell"
+
+func (widget *Widget) initializeKeyboardControls() {
+	widget.SetKeyboardChar("/", widget.ShowHelp, "Show/hide this help window")
```

Comment:
Can you take a look at the JIRA module and add appropriate keyboard commands? `r/j/k` would be appreciated for consistency.